volim.html

```diff
@@ -100,5 +100,21 @@
 			<p class="openhour-lesson__dance">Tab Dancing : 19:00 - 23.00</p>
 		</div>
 	</div>
```

Comment:
Looks like your markup doesn't align properly. Please, pretiffy that.

Reply:
![image](https://user-images.githubusercontent.com/61840948/103297112-4bf90b00-4a00-11eb-8274-cf850fc071fc.png)